src/app/static/src/app/components/Stepper.vue

```diff
@@ -78,21 +84,24 @@
             }
         },
         computed: {
-            baselineComplete: function() {
-              return this.$store.getters['baseline/complete']
+            baselineComplete: function () {
+                return this.$store.getters['baseline/complete']
             },
-            surveyAndProgramComplete: function() {
+            surveyAndProgramComplete: function () {
                 return this.$store.getters['surveyAndProgram/complete']
             },
-            complete: function(): CompleteStatus {
+            complete: function (): CompleteStatus {
                 return {
                     1: this.baselineComplete,
                     2: this.surveyAndProgramComplete,
                     3: this.surveyAndProgramComplete, // for now just mark as complete as soon as it's ready
                     4: false,
                     5: false
                 }
-            }
+            },
+            ...mapState<RootState>({
+                ready: state => state.surveyAndProgram.ready && state.baseline.ready
```

Comment:
I suppose ready could be a getter on the root state - to check the ready state of all the component states. Maybe not worth it if the stepper is the only thing that needs to use it.

Reply:
I think this is the only component that needs it